File: years/2017/elixir/test/aoc/day_09_test.exs
defmodule AoC.Day09.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert AoC.Day09.part_1(["../data/09.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert AoC.Day09.part_2(["../data/09.txt"]) == 1
    end
  end
end

File: lang/clojure/skeleton/test/day_09_test.clj
(ns day-09-test
  (:require
   [clojure.test :refer [deftest is testing]]
   [day-09 :refer [part1 part2]]))

(deftest day-09-test
  (testing "sample data"
    (is (= 0 (part1 "../data/09-sample.txt")))
    (is (= 0 (part2 "../data/09-sample.txt")))))
